.github/workflows/devbuilds.yml

```diff
@@ -36,28 +69,81 @@ jobs:
-    - name: Setup Github credentials
-      uses: fusion-engineering/setup-git-credentials@v2
-      with:
-        credentials: ${{secrets.GIT_CREDENTIALS}}      
-    - name: Update dev_build tag and release
-      run: |
-        git config --global user.email lorenzesch@hotmail.com
-        git config --global user.name LorenzE
-        git tag dev_build -f -a -m "Development Builds"
-        git push -f --tag
-    - name: Deploy binaries with dev_build release at Github
+        tar cfvz ../mne-cpp-linux-x86_64.tar.gz ./*
+    - name: Deploy binaries with dev_build release on Github
       uses: svenstaro/upload-release-action@v1-release
       with:
         repo_token: ${{ secrets.GITHUB_TOKEN }}
         file: mne-cpp-linux-x86_64.tar.gz
         asset_name: mne-cpp-linux-x86_64.tar.gz
         tag: dev_build
         overwrite: true
+
+  MacOS:
+    runs-on: macos-latest
+    needs: UpdateDevTag
+
+    steps:
+    - name: Clone repository
+      uses: actions/checkout@v1
+    - name: Install Qt
+      uses: jurplel/install-qt-action@v2
+      with:
+        version: 5.14.0
+        modules: qtcharts
+    - name: Configure and compile MNE-CPP
+      run: |        
+        qmake -r MNECPP_CONFIG+=noTests MNECPP_CONFIG+=noExamples
+        make -j2
+    - name: Package binaries
+      run: |
+        # Creating archive of all macos deployed applications
+        tar cfvz mne-cpp-macos-x86_64.tar.gz bin/mne_scan.dmg bin/mne_browse.dmg bin/mne_analyze.dmg
+    - name: Deploy MNE Scan binaries with dev_build release on Github
+      uses: svenstaro/upload-release-action@v1-release
+      with:
+        repo_token: ${{ secrets.GITHUB_TOKEN }}
+        file: mne-cpp-macos-x86_64.tar.gz
+        asset_name: mne-cpp-macos-x86_64.tar.gz
+        tag: dev_build
+        overwrite: true
+
+  Win:
+    runs-on: windows-2016
+    needs: UpdateDevTag
+
+    steps:
+    - name: Clone repository
+      uses: actions/checkout@v1      
+    - name: Install Qt
+      uses: jurplel/install-qt-action@v2
+      with:
+        version: 5.14.0
+        arch: win64_msvc2017_64
+        modules: qtcharts
+    - name: Install jom
+      run: choco install jom
+    - name: Configure and compile MNE-CPP
+      run: |        
+        cmd.exe /c "call `"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\amd64\vcvars64.bat`" && set > %temp%\vcvars.txt"
+        Get-Content "$env:temp\vcvars.txt" | Foreach-Object { if ($_ -match "^(.*?)=(.*)$") { Set-Content "env:\$($matches[1])" $matches[2] } }
+        qmake -r MNECPP_CONFIG+=noTests
+        jom -j2
+    - name: Package binaries
+      run: |
+        7z a mne-cpp-windows-x86_64.zip ./bin
+    - name: Deploy binaries with dev_build release on Github
+      uses: svenstaro/upload-release-action@v1-release
+      with:
+        repo_token: ${{ secrets.GITHUB_TOKEN }}
+        file: mne-cpp-windows-x86_64.zip
+        asset_name: mne-cpp-windows-x86_64.zip
+        tag: dev_build
+        overwrite: true
```

Comment:
Nicely readable - not clear to me if deploy still uses the Qt deploy commands

Reply:
What do you mean by Qt deploy commands?